.github/BOTMETA.yml

```
@@ -1460,6 +1460,10 @@ files:
     maintainers: vbotka
   docs/docsite/rst/filter_guide_abstract_informations_merging_lists_of_dictionaries.rst:
     maintainers: vbotka
+  docs/docsite/rst/filter_guide-abstract_informations-types.rst:
```

Please use underscores:
```suggestion
  docs/docsite/rst/filter_guide-abstract_informations_types.rst:
```
Same below.

---

Why is this needed?

The name of the file is a concatenation of three headings' levels:

* filter_guide
* abstract_informations
* type

The concatenation by dash "-" keeps this information. I've already created a couple of files this way. Should all of them be renamed?

You kept the first dash. Do you think this one should be replaced too?

For the record: This name is used in filter_guide_abstract_informations.rst

---

I didn't catch the dashes in the previous PR. Generally all filenames in docs/docsite/rst/ use underscores instead of dashes. It would be great to keep it that way.